test/integration/voldemort/performance/benchmark/Benchmark.java

```diff
@@ -642,6 +651,18 @@ public static void main(String args[]) throws IOException {
               .withRequiredArg()
               .describedAs(LOCAL_SERVER_PROPERTIES)
               .ofType(String.class);
+        parser.accepts(WORKLOAD_CLASS, "workload class (full name)")
+              .withRequiredArg()
+              .describedAs(WORKLOAD_CLASS)
+              .ofType(String.class);
```

Comment:
Can you please add help message about what are the possible options of the Workload class ? I believe exposing the class is probably not convenient and sort of gives an impression that user can supply their workload. Not a big deal, but would be better if it can be renamed as WORKLOAD_TYPE and just let the type be former or the trace replay.